public/datasets/erin_candy_preferences.json

```diff
@@ -1,4 +1,5 @@
 {
+  "defaultLabelColumn": "name",
```

Comment:
I think the default label column in this case is `delicious?`;  the model won't predict a candy's name very well.